xero_accounting.yaml

```diff
@@ -23393,6 +23400,14 @@ components:
       schema:
         type: string
         format: date
+    pageSize:
+      in: query
+      name: pageSize
+      x-snake: page_size

```

Comment:
This x-snake, what is the relevance of it? 
Note that the purpose of this PR is to add the pagesize query parameter to api-explorer.

Reply:
I've removed that x-snake.